src/modality-specific-files/motion.md

```diff
@@ -0,0 +1,195 @@
+
+## Channels description (`*_channels.tsv`)
+
+{{ MACROS___make_filename_template(
+"raw",
+datatypes=["motion"],
+suffixes=["channels"])
+}}
+
+This file is REQUIRED as it makes it easy to browse or query over larger collections of datasets.
+The REQUIRED columns are channel `name`, `component`, `type`, `tracked_point` and `units`.
+Any number of additional columns MAY be added to provide additional information about the channels.
+The `*_tracksys-<label>_channels.tsv` file SHOULD give additional information about individual recorded channel, some of which my not be found summarized in `*_motion.json`.
+
+The columns of the channels description table stored in `*_channels.tsv` are:
+
+{{ MACROS___make_columns_table("motion.motionChannels") }}
+
+### Restricted keyword list for channel component
+
+Restricted keyword list for column `component`.
+When using quaternions to represent orientations, the axial components that corresponds to the three spatial axes MUST be specified as "quat_x", "quat_y", "quat_z", and the non-axial component as "quat_w".
+
+| **Keyword** | **Description**                                                                                                                                             |
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| x           | Position along the X-axis, or rotation about the X-axis among the Euler angles that represent the orientation, or magnetic field strength along the X-axis. |
+| y           | Position along the Y-axis or rotation about the Y-axis among the Euler angles that represent the orientation, or magnetic field strength along the Y-axis.  |
+| z           | Position along the Z-axis or rotation about the Z-axis among the Euler angles that represent the orientation, or magnetic field strength along the Z-axis.  |
+| quat_x      | Quaternion component associated with the X-axis.                                                                                                            |
+| quat_y      | Quaternion component associated with the Y-axis.                                                                                                            |
+| quat_z      | Quaternion component associated with the Z-axis.                                                                                                            |
+| quat_w      | Non-axial quaternion component.                                                                                                                             |
+| n/a         | Channels that have no corresponding spatial axis.                                                                                                           |
+
+### Restricted keyword list for channel type
+
+Restricted keyword list for column `type` in alphabetic order.
+Note that upper-case is REQUIRED:
```

Comment:
not a blocker for merging this PR, but before release, we must clarify how channel types are shared between modalities. I have opened a new issue for this:

- https://github.com/bids-standard/bids-specification/issues/1436